CMakeLists.txt

```diff
@@ -67,13 +51,21 @@ if (BUILD_EXAMPLES)
 	add_subdirectory(examples)
 endif()
 
+# library interface
+file(GLOB_RECURSE headers_library "${TAPKEE_INCLUDE_DIR}/*.hpp")
+add_library(tapkee_library INTERFACE)
+target_include_directories(tapkee_library INTERFACE ${TAPKEE_INCLUDE_DIR} "${EIGEN3_INCLUDE_DIR}")
+
 if (ARPACK_FOUND)
 	target_link_libraries(tapkee arpack)
+	include_directories(${OPENGL_INCLUDE_DIR} ${GLEW_INCLUDE_DIRS})
```

Comment:
Hmm, I did that some time ago, I thought it was that arpack was proposing option to use opengl, but now that I've checked, doesn't seems so. I'll remove that !